src/zope/app/apidoc/utilities.rst

```diff
@@ -185,7 +187,7 @@ One can also pass functions
 
 and even methods. If a method is passed in, its class path is returned.
 
-  >>> utilities.getPythonPath(Sample.sample)
+  >>> utilities.getPythonPath(Sample().sample)
```

Comment:
It pretty much had to be to make it work on Python 2 and 3, given the absence of unbound methods now. (But again, open to suggestion.)